### PR TITLE
Bump app-emulation/sen to 0.6.0

### PR DESCRIPTION
**Pull request by ntnn**
Also adds new test dependency `dev-python/flexmock` to the tree.

### CODE REVIEW

File: dev-python/flexmock/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
```
Comment (monsieurp) [inline dev-python/flexmock/metadata.xml]: Make sure to add the Proxy Maintainer group to the list of maintainers as well.

File: dev-python/flexmock/flexmock-0.10.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+
+DEPEND="
+	"
+RDEPEND="${DEPEND}"
+BDEPEND="${DEPEND}
```
Comment (monsieurp) [inline dev-python/flexmock/flexmock-0.10.2.ebuild]: `BDEPEND` is hardly ever used across the Portage tree. Please nix it in favour of `DEPEND`.

File: app-emulation/sen/sen-0.6.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+
+DEPEND="dev-python/urwid[${PYTHON_USEDEP}]
+	dev-python/urwidtrees[${PYTHON_USEDEP}]
+	dev-python/docker-py[${PYTHON_USEDEP}]
+	"
+RDEPEND="${DEPEND}"
+BDEPEND="${DEPEND}
```
Comment (monsieurp) [inline app-emulation/sen/sen-0.6.0.ebuild]: Ditto.

File: dev-python/flexmock/flexmock-0.10.2.ebuild (hunk 2 above)
Comment (mgorny) [reply dev-python/flexmock/flexmock-0.10.2.ebuild]: `BDEPEND` is future EAPI 7 variable name, please don't use it.

**Comment by ntnn** [comment]
@monsieurp Thanks, I've updated the PR and also added proxy-maint to the sen metadata.

**Comment by mgorny** [comment]
Thanks! Looks good now, will merge.

**Pull request closed without merging**